CHANGELOG.md

```diff
@@ -84,7 +86,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   open in a new window, unless the link points to a CyberArk docs site.
   [cyberark/conjur-oss-suite-release#199](https://github.com/cyberark/conjur-oss-suite-release/issues/199)
 
-[Unreleased]: https://github.com/cyberark/conjur-oss-suite-release/compare/v1.18.0+suite.1...HEAD
+[Unreleased]: https://github.com/cyberark/conjur-oss-suite-release/compare/v1.18.4+suite.1...HEAD
+[v1.18.4+suite.1]: https://github.com/cyberark/conjur-oss-suite-release/compare/v1.15.0+suite.1...v1.18.4+suite.1
 [v1.18.0+suite.1]: https://github.com/cyberark/conjur-oss-suite-release/compare/v1.15.0+suite.1...v1.18.0+suite.1
```

Comment:
A couple of these links need to be updated. These links are comparing their version against `v1.15.0+suite.1` - instead, they should compare against the version immediately prior:
- `v1.18.0+suite.1`
- `v1.18.4+suite.1`